pkg/reconciler/taskrun/entrypoint/entrypoint.go

```diff
@@ -141,12 +147,13 @@ func RedirectStep(cache *Cache, stepNum int, step *v1alpha1.Step, kubeclient kub
 	if stepNum == 0 {
 		step.VolumeMounts = append(step.VolumeMounts, downwardMount)
 	}
+	step.VolumeMounts = append(step.VolumeMounts, logCfgMount)
```

Comment:
Can we avoid mounting this if we're not using the logging entrypoint runner? Do we care?